rasa_nlu/tokenizers/__init__.py

```diff
@@ -0,0 +1,19 @@
+def tokenizer_from_name(name, language):
+    from rasa_nlu.tokenizers.whitespace_tokenizer import WhitespaceTokenizer
+    from rasa_nlu.tokenizers.mitie_tokenizer import MitieTokenizer
+    from rasa_nlu.tokenizers.spacy_tokenizer import SpacyTokenizer
+
+    if name == MitieTokenizer.name:
+        return MitieTokenizer()
+    elif name == SpacyTokenizer.name:
+        import spacy
+        nlp = spacy.load(language, parser=False, entity=False)
```

Comment:
The server won't use this method. It will lookup the name of the component in the registry, get the class from there and then call the load method on that class.

This method is only used for the luis data conversion.